scripts/performance/memory-performance/compute-memory-performance.js

```diff
@@ -1,6 +1,7 @@
 const puppeteer = require('puppeteer')
 const { fetchPR, LOCAL_BRANCH } = require('../../lib/git-utils')
-const NUMBER_OF_RUNS = 40 // Rule of thumb: this should be enough to get a good average
+const NUMBER_OF_RUNS = 30 // Rule of thumb: this should be enough to get a good average
+const BATCH_SIZE = 2
```

Comment:
❓ **Question**: Why just 2 ? Couldn't we increase it more ?

Reply:
On the CI instance, more process doesn't bring better performances.